<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SyncEdit</class>
 <widget class="QDialog" name="SyncEdit">
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Master store</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Slave store</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Sync type</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QFrame" name="profileFrame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QRadioButton" name="noneProfile">
              <property name="text">
               <string>None</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="biDirProfile">
              <property name="text">
               <string>Bi-directional sync</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="backupProfile">
              <property name="text">
               <string>Backup / Mirror</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="followProfile">
              <property name="text">
               <string>Follow</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="manualProfile">
              <property name="text">
               <string>Manual configuration</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="masterStore"/>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="slaveStore"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="manualGroup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Manual configuration</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Forward sync</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Backward sync</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="forwardType">
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Fast forward</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pick newest</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Three-way merge</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="backwardType">
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Fast forward</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pick newest</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Three-way merge</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="description">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Description</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SyncEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SyncEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manualProfile</sender>
   <signal>toggled(bool)</signal>
   <receiver>manualGroup</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>masterStore</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SyncEdit</receiver>
   <slot>storesSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slaveStore</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SyncEdit</receiver>
   <slot>storesSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>storesSelected()</slot>
 </slots>
</ui>
